src/tests/routes/parks.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
- [ ] Please 🙏🏽 remove commented code while it is not to help to understand your code, remember YAGNI.